syntax = "proto3";

package robodog;

message MockMsg {

  int32 mockValue = 1;
}
